iamtest/mockpermissiontester.go

```diff
@@ -0,0 +1,84 @@
+package iamtest
+
+import (
+	"context"
+	"sync"
+
+	"go.einride.tech/aip/resourcename"
+	"go.einride.tech/iam/iamresource"
+
+	iamv1 "go.einride.tech/iam/proto/gen/einride/iam/v1"
+)
+
+// MockPermissionTester is a mock permission tester.
+type MockPermissionTester struct {
+	mu                 sync.Mutex
+	allowAll           bool
+	allowedPermissions []allowedPermission
```

Comment:
grantedBindings?

Reply:
I'm using terminology from the Google IAM documentation here with "allowed permissions"